app/listeners/app.init/initPassport.js

```diff
@@ -22,7 +22,7 @@ function initPassport (app) {
             .send(userData)
             .end(function (err, resp) {
             if (err) {
-                return done (err,false);
+                return done (null,false);
```

Comment:
I actually think that was correct, don't we want to return the error if an error occurs?  Maybe, I am misunderstanding.